content/browser/devtools/devtools_agent_host_impl.cc

```diff
@@ -143,6 +143,7 @@ const char DevToolsAgentHost::kTypeOther[] = "other";
 const char DevToolsAgentHost::kTypeAuctionWorklet[] = "auction_worklet";
 const char DevToolsAgentHost::kTypeAssistiveTechnology[] =
     "assistive_technology";
+const char DevToolsAgentHost::kTypeBrowserUI[] = "browser_ui";
```

Comment:
Just a note, without https://chromium-review.googlesource.com/c/devtools/devtools-frontend/+/7018087, we will lose the ability to debug the top level frame with DevTools. Try to see if chaning it to `="page"` will allow that to work while still being able to run WPT tests.